google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -342,6 +342,23 @@ protected InflightBytesLimitExceededException(String writerId, long currentLimit
           currentLimit);
     }
   }
+  /**
+   * Input Json data has unknown field to the schema of the JsonStreamWriter. User can each turn on
+   * IgnoreUnknownFields option on the JsonStreamWriter, or if they don't want the error to be
```

Comment:
What will happen to backend if the user keep pass in json that bypass this check?